source/manual/incorrect-content-in-search-or-navigation.html.md

```diff
@@ -63,6 +63,7 @@ reindexing via a rake task in the publishing app.
 
 Try republishing the content.
 
-#### Unpublished content is still showing up in search
+#### Unpublished or outdated content is still showing up in search
```

Comment:
I'm not sure about "outdated", as content can be outdated but still supposed to be in search (it should just be up-to-date).

Maybe "Unpublished, deleted, or redirected content is still showing up in search"

Reply:
very good point, thank you